js/packages/teams-ai/src/authentication/MessagingExtensionAuthenticationBase.ts

```diff
@@ -0,0 +1,95 @@
+import { ActivityTypes, InvokeResponse, TokenResponse, TurnContext } from 'botbuilder';
+import { MessagingExtensionsInvokeNames } from '../MessageExtensions';
+
+/**
+ * @internal
+ */
+export abstract class MessagingExtensionAuthenticationBase {
```

Comment:
Isn't the naming `MessageExtension`? Or is it different for authentication?

Reply:
[Documentation](https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/add-authentication) would indicate the correct term is `MessageExtension`:
![image](https://github.com/microsoft/teams-ai/assets/14900841/510ca53b-cab9-4346-9ec9-755b61a5f88b)

Could we rename away from `MessagingExtension`?

Reply:
Uhhh nevermind. Botbuilder has imports that are also `MessagingExtension` instead of `MessageExtension` so there's no way to fully fix the inconsistency.